charts/jenkins/values.yaml

```diff
@@ -79,6 +79,7 @@ jenkins:
     - pipeline-stage-tags-metadata
     - pipeline-stage-view
     - plain-credentials
+    - powershell
```

Comment:
is it needed? the docs say it doesn't integrate with pipeline? https://plugins.jenkins.io/powershell/

Reply:
Powershell is supported negatively in the Pipeline Durable Task plugin: https://jenkins.io/blog/2017/07/26/powershell-pipeline/ . So I am not sure why we need this plugin TBH. @slide ?

Reply:
You are correct.